### ISSUE TITLE
Error in creating Unity view

### ISSUE THREAD

**Issue by hyperms** [juicycleff/flutter-unity-view-widget#266]
In some Android devices when creating a Unity view, Unity runs in the background but the Unity view is not displayed.
After pressing the home key and returning to the application, the view is displayed.

**Comment by hyperms** [comment]
device info of two Android devices on which this issue occurred
![IMG_20201203_122601_300](https://user-images.githubusercontent.com/72021670/100987358-ad45cf80-3563-11eb-901c-2bff5dd24943.jpg)
![IMG_20201203_122558_622](https://user-images.githubusercontent.com/72021670/100987395-b636a100-3563-11eb-854b-e8047328ee64.jpg)

**Comment by juicycleff** [comment]
Hello, I have tested the example on an emulator and it works just fine. I can't debug it. Maybe you can try your game directly from unity on the devices and see if you get thesame result.

**Comment by hyperms** [comment]
Yes, we tested our game on these devices and it ran without any problems
But this problem occurs when we outputs for flutter
Could the problem be with these settings?

![IMG_20201207_004611_971](https://user-images.githubusercontent.com/72021670/101292594-bb2a7780-3825-11eb-89ef-e3a64401b1e2.jpg)

Are these settings set in the editor that outputs for flutter?
